src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') CODE_SMELL
Complete the task associated to this TODO comment.(AWBEzrmbI9pSNYZJmpeF)

[ISSUE_KEY]: AWBEzrmbI9pSNYZJmpeF